### ISSUE TITLE
Video Tutorial on how to build a USBValve 

### ISSUE THREAD

**Issue by Tz1rf** [cecio/USBvalve#38]
I thought I would post this here. I made a video tutorial on how to build a USBValve. Anyone should be able to follow this tutorial and build one at home. Feel free to post/share it and I hope those interested in this project find it helpful! [Youtube Video](https://youtu.be/7ymk8hD7-Hc)

**Comment by cecio** [comment]
Wow @Tz1rf awesome work!
If it's ok for you I'd like to link it in the README file of the project.
Thanks a lot!

**Comment by Tz1rf** [comment]
Yes you can link where ever you would like. You don't need my permission to distribute the video. Feel free to share it/post it anyplace you want.

**Comment by cecio** [comment]
Thanks!!

**Comment by LiPolymer** [comment]
@Tz1rf May I re-upload  your video on [BiliBili](https://www.bilibili.com/)?

YouTube has been banned in China,so that's pretty difficult to access the original link of your amazing video for Chinese users. :(

I'll mark the original link in the description. ;)

Thank you for your video :D

**Comment by Tz1rf** [comment]
I would prefer that you don't. Re-uploading my video is not the same as sharing/posting it using the youtube link.

**Comment by LiPolymer** [comment]
> I would prefer that you don't. Re-uploading my video is not the same as sharing/posting it using the youtube link. 

Thanks for your reply, I will not re-upload this video :D

**Comment by Tz1rf** [comment]
I created a video showing how to flash the firmware to the Valve as well as how to use the device. https://youtu.be/Tp8xvrlqxUY

**Comment by cecio** [comment]
Awesome! Thank you very much! I'm going to link also this one in the README. 
Thanks!
